custom_dc/README.md

```diff
@@ -278,7 +295,8 @@ gcloud run deploy datacommons \
   --image us-central1-docker.pkg.dev/$PROJECT_ID/datacommons/website-compose:$CUSTOM_DC_TAG \
   --add-cloudsql-instances=<project>:<region>:dc-graph \
   --set-env-vars="$env_vars" \
-  --port 8080
+  --port 8080 \
+  --network default
```

Comment:
Should we make the network configurable with `default` as default?

Reply:
Updated this command ( & fixed the command to add `beta` to the cloud run.